site/accounts/staywelcome.php

```diff
@@ -0,0 +1,11 @@
+<?php if(!defined('KIRBY')) exit ?>
+
+username: staywelcome
+password: >
+  $2a$10$LhoBWrjK9kPdmbD7/CsZVOopiTTpDRAiLCMrtucCtL5.P0O.OyIYm
```

Comment:
@dhabersack is it a problem that the password (or rather its hash) is made public here?
or are we on the safe side, since nobody is able to guess the password from the hash here?
should we `.gitignore` this file?